source/includes/administration/_email_settings.md

```diff
@@ -0,0 +1,171 @@
+## Email Settings
+
+<!-------------------- GET EMAIL SETTINGS -------------------->
+
+### Retrieve email settings
+
+`GET /email_settings/:id`
+
+```shell
+# Retrieve knowledge base
+curl "https://cloudmc_endpoint/rest/brandings/dc30f9a6-1edc-11eb-92b6-0242ac120003" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "mailgunPrivateKey": "VALIDPRIVATEKEY",
+    "smtpHost": "valid.host",
+    "smtpFromAddress": "valid.address@domain.com",
+    "smtpUsername": "valid.username@gmail.com",
+    "organization": {
+      "id": "fb1b2caf-4ca4-4c45-b094-faa84bdabc47"
+    },
+    "smtpPort": 2,
+    "mailgunDomain": "valid.mailgun.domain",
+    "smtpPassword": "coolPassword",
+    "smtpUseSsl": false,
+    "id": "dc30f9a6-1edc-11eb-92b6-0242ac120003"
+  }
+}
+```
+Retrieve the email settings associated to the id.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of email settings.
```

Comment:
The configured email settings' id? I feel like "The id of email settings." sounds weird